dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -242,7 +247,13 @@ public String getAlternateInForwardStrand() {
             throw new IllegalArgumentException("Neither the HGVS_C nor HGVS_T strings are defined");
         }
 
-        if (orientation.equals(Orientation.FORWARD)) {
+        return complementAlleleIfNecessary(allele, orientation);
+    }
+
+    private String complementAlleleIfNecessary(String allele, Orientation orientation) {
```

Comment:
It is not just complement but reverse complement. I wonder if it could be better to call it `getAlleleInForwardStrand`, because that is what it does.